### PR TITLE
Added docker

### PR DESCRIPTION
**Pull request by HovorukhaBohdan**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=Root-159363
+MYSQLDB_DATABASE=spring-boot
+MYSQLDB_LOCAL_PORT=3308
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (d1sam) [inline .env]:
```suggestion
DEBUG_PORT=5005

```

File: .env [hunk 2]
```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=Root-159363
```
Comment (d1sam) [inline .env]:
```suggestion
MYSQLDB_ROOT_PASSWORD=***********
```

Comment (d1sam) [reply .env]: please don't push real password to GitHub

**Review by d1sam — CHANGES_REQUESTED** [review]
good job, but please make some minor changes :)

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**